egs/heroico/s5/local/prepare_dict.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash -u
+
+# Copyright 2017 John Morgan
+# Apache 2.0.
+
+set -o errexit
+
+[ -f path.sh ] && . path.sh  # Sets the PATH to contain necessary executables
```

Comment:
this should be probably written as 
```
[ -f ./path.sh ] && . ./path.sh
```
otherwise the first path.sh on the PATH would be sourced.